### PR TITLE
[New] Add hola_player w/ npm auto-update

### PR DESCRIPTION
**Pull request by sufuf3**
Pull request for issue: #10131
Related issue(s): #10133 #11170

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/hola/hola_player
 * Official website (optional, not the repository): http://holacdn.com/player
 * NPM package url (optional): https://www.npmjs.com/package/@hola.org/hola_player
 * License and its reference: ISC
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 27
   - Count of stars: 17
   - Count of forks: 10
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 579
   - Downloads in the last month: 4099

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@cdnjs/intern3 
Please help me review this PR, Thank you.

File: ajax/libs/hola_player/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "license": "ISC",
+  "keywords": [
+    "flash",
+    "html5",
+    "player",
+    "video",
+    "videojs"
+  ],
+  "homepage": "http://holacdn.com/player",
+  "author": "Steve Heffernan",
```
Comment (extend1994) [inline ajax/libs/hola_player/package.json]: How did you know who is the author?

File: ajax/libs/hola_player/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "filename": "hola_player.js",
+  "license": "ISC",
+  "keywords": [
+    "flash",
+    "html5",
+    "player",
+    "video",
+    "videojs"
+  ],
+  "homepage": "http://holacdn.com/player",
```
Comment (extend1994) [inline ajax/libs/hola_player/package.json]: Please use https for more secure browsing.

Comment (sufuf3) [inline ajax/libs/hola_player/package.json]: I'd like to.
But there is a chain issue on this website.
I don't know is it good to use `ssl`.
ref: https://www.ssllabs.com/ssltest/analyze.html?d=holacdn.com&s=184.73.238.209

@pvnr0082t could you suggest us?

File: ajax/libs/hola_player/package.json (hunk 1 above)
Comment (sufuf3) [inline ajax/libs/hola_player/package.json]: @extend1994 I'm sorry for filter in wrong author. 
The author is Bahaa Aidi.  ref: https://github.com/hola/hola_player/graphs/contributors?from=2016-12-04&to=2016-12-11&type=c
I'll modify it when I get the answer of homepage url.

File: ajax/libs/hola_player/package.json (hunk 2 above)
Comment (sufuf3) [inline ajax/libs/hola_player/package.json]: @pvnr0082t could you suggest us?

Comment (pvnr0082t) [inline ajax/libs/hola_player/package.json]: I think we don't use `https` here.

**Comment by sufuf3** [comment]
@extend1994 I modified the `author`
Please help me review this PR, Thank you.
As @pvnr0082t suggested I didn't modify it.

**Comment by sufuf3** [comment]
@cdnjs/library-reviewer 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@hare1039 @qwe19272375 @pvnr0082t @maruilian11 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@hare1039 @qwe19272375 @pvnr0082t @maruilian11
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@PeterDaveHello 
Please help me review this PR, Thank you.
I can't require your help to review this PR because I got the message `You can't perform that action at this time.`.

**Comment by PeterDaveHello** [comment]
Ping @maruilian11

**Comment by sufuf3** [comment]
@maruilian11 
1. over 500 downloads in the last month. https://www.npmjs.com/package/@hola.org/hola_player
2. the old PR is in this link: https://github.com/cdnjs/cdnjs/pulls?utf8=%E2%9C%93&q=is%3Aopen%20is%3Apr%20sort%3Aupdated-asc%20s
3. This lib reach the minimum popularity requirement.
4. Yes

Thank you.

**Comment by sufuf3** [comment]
@maruilian11 @pvnr0082t 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@pvnr0082t I added v1.0.39 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@maruilian11 @pvnr0082t
Please help me review this PR, Thank you.

File: ajax/libs/hola_player/package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hola/hola_player.git"
+  },
+  "npmName": "@hola.org/hola_player",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "hola_player*",
```
Comment (maruilian11) [inline ajax/libs/hola_player/package.json]: Let's use exclusive rule for dist folder. Like `**/!(*.zip)`.

**Comment by sufuf3** [comment]
@maruilian11 I modified the `filemap` and updated to v1.0.67.
Please help me review this PR, Thank you.

**Pull request merged**